tt_metal/impl/dispatch/command_queue.cpp

```diff
@@ -1311,16 +1317,19 @@ void CommandQueue::run_worker() {
 }
 
 void CommandQueue::run_command(const CommandQueueInterface& command) {
-    tt::log_trace(tt::LogDispatch, "CQ{} received {} in {} mode", cq_id, command.type, async_mode() ? "ASYNC" : "PASSTHROUGH");
+    log_trace(tt::LogDispatch, "CQ{} received {} in {} mode", cq_id, command.type, async_mode() ? "ASYNC" : "PASSTHROUGH");
     if (async_mode()) {
```

Comment:
removed tt:: scoping from log_trace but not LogDispatch?

Reply:
Yeah my bad, will clean up.

There is actually some inconsistency in `log_trace` implementation that led to the removal -- it's a tt namespace function in debug build and a global scope macro in release build... So I just took it out to get compile working without thinking about the consistency

Reply:
filed issue https://github.com/tenstorrent-metal/tt-metal/issues/5413

Also logging functions are worst perf vs. logging macros, we made that switch in buda, I added that suggesting there too. Plus why use log_trace vs. log_debug